<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfacciaAdmin</class>
 <widget class="QWidget" name="InterfacciaAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:rgb(113, 113, 113);
}

QPushButton
{
	background-color:qlineargradient(spread:pad, x1:0.474, y1:1, x2:1, y2:0, stop:0.507177 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 127, 255));
	font: bold 14pt &quot;Kinnari&quot;;
	color: rgb(255, 170, 0);
	border-width: 2px;
	border-color:rgb(255, 85, 0);
	border-style: solid;
	border-radius: 7;
}

QPushButton:hover
{
	background-color:qlineargradient(spread:pad, x1:0.474, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.507177 rgba(255, 170, 0, 255)); 
	color: rgb(0, 0, 0);
}


QPushButton:hover:!pressed
{
	background-color:qlineargradient(spread:pad, x1:0.244, y1:1, x2:0.837, y2:0.227273, stop:0 rgba(255, 255, 127, 255), stop:0.507177 rgba(0, 0, 0, 255));
	color: rgb(255, 170, 0);
}

#AdminLabel
{
	font: bold italic 32pt &quot;Kinnari&quot;;
	color: rgb(255, 85, 0)
}

#GestisciLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
}

#RicercaLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
}

#NuovoUtenteLabel
{
	font: bold 14pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127)
}

QLabel
{
	font: bold 14pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127)
}

QLineEdit
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	selection-color: rgb(0, 0, 0);
	selection-background-color: rgb(255, 170, 0);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

QRadioButton
{
	font: bold 14pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color:rgb(57, 57, 57);
}

QTableWidget
{
	color:rgb(255, 255, 255);
	background-color:rgb(35, 35, 35);
	gridline-color:rgb(255, 85, 0);
}

QHeaderView::section 
{
	padding:0;
	height: 32px;
	width:18px;
	font:bold 14pt &quot;Kinnari&quot;;
	color:rgb(255, 85, 0);
	background-color:rgb(118, 118, 118);
}

QTableWidget::item:selected
{ 
	color: rgb(0, 0, 0);
	background-color: rgb(255, 170, 0);
 }

QTableWidget QTableCornerButton::section 
{
	background-color:rgb(255, 85, 0);
}

QGroupBox
{
	background-color:rgb(57, 57, 57);
	border-width: 2px;
	border-color:rgb(255, 85, 0);
	border-style: solid;
	border-radius: 7;

}

#UsernameLabel
{
	background-color:rgb(57, 57, 57);
}

#Username1Label
{
	background-color:rgb(57, 57, 57);
}

#Username2Label
{
	background-color:rgb(57, 57, 57);
}

#NuovoUtenteLabel
{
	background-color:rgb(57, 57, 57);
}

#NomeLabel
{
	background-color:rgb(57, 57, 57);
}

#CognomeLabel
{
	background-color:rgb(57, 57, 57);
}

#FrecciaLabel
{
	background-color:rgb(57, 57, 57);
}

#Freccia2Label
{
	background-color:rgb(57, 57, 57);
}

#ChiudiButton
{
	font: bold 16pt &quot;Kinnari&quot;;
}
</string>
  </property>
  <widget class="QLabel" name="AdminLabel">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>0</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>840</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>100</y>
     <width>350</width>
     <height>180</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="Username2Label">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>20</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Usename</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NuovoUtenteLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Nuovo Utente</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="FrecciaLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-------&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EliminaLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="EliminaUtenteButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Elimina</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InserisciNuovoUtenteButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Inserisci</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>100</y>
     <width>350</width>
     <height>180</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="CambiaLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="BasicRadioButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Basic</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="BusinessRadioButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Business</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ExecutiveRadioButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Executive</string>
    </property>
   </widget>
   <widget class="QLabel" name="Username1Label">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>20</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Usename</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="CambiaTipoAccountButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>190</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Cambia Tipo Account</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="GestisciLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestisci Utenti Iscritti</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>345</y>
     <width>370</width>
     <height>200</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="UsernameRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>20</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="NomeCognomeRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>95</y>
      <width>20</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="UsernameLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>35</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NomeLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>95</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="CognomeLineEdit">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>95</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CercaButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Cerca</string>
    </property>
   </widget>
   <widget class="QLabel" name="UsernameLabel">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>5</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Usename</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NomeLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>65</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="CognomeLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>65</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cognome</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>840</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>100</y>
     <width>10</width>
     <height>195</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="VisualizzaUtentiIscrittiButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>566</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Visualizza Tutti Gli Utenti </string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChiudiButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>620</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Chiudi</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="Table">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>438</width>
     <height>270</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>104</number>
   </attribute>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cognome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Username</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tipo Utente</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="FrecciaLabel_2">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>570</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>-------&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="RicercaLabel">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>305</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ricerca</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
